WcaOnRails/app/helpers/registrations_helper.rb

```diff
@@ -36,7 +36,7 @@ def please_sign_in(message_key, comp, args = {})
   def registration_date_and_tooltip(competition, registration)
     if @competition.using_stripe_payments?
       [registration.last_payment_date&.to_date || I18n.t('registrations.list.not_paid'),
-       I18n.t('registrations.list.payment_requested_on', date: registration.created_at)]
```

Comment:
nit - is there a better way to handle this without a really ugly ternary?

Reply:
The only alternative I can think of is to do something like this in the `if`:
```ruby
tooltip_message = if registration.last_payment_date
                    I18n.t('registrations.list.payment_completed_on', date: registration.last_payment_date)
                  else
                    I18n.t('registrations.list.payment_requested_on', date: registration.created_at)
                  end
```
and return the tooltip message.
But honestly the ternary doesn't look that bad to me.

Reply:
I've seen uglier ternaries by all means :D
What you can do is try to make it look pretty by changing the indentation, but that is a super-nitpick and should not block this PR. Using a ternary expression per se is totally fine for this code.